docs/integrations/query.mdx

```diff
@@ -227,16 +227,58 @@ FIXME: Update example
 
 See [a working example](https://codesandbox.io/s/04mepx) to learn more.
```

Comment:
We have one more `FIXME` here. If you can help fixing it,  we'd be more than happy.
Otherwise, I can just merge it as is.

Reply:
sure, i fixed

Reply:
thanks so much!!!